python/ray/data/_internal/datasource/parquet_datasource.py

```diff
@@ -195,6 +195,7 @@ def __init__(
                 ray.get_runtime_context().get_node_id(), soft=False
             )
 
+        self.unresolved_path = paths
```

Comment:
nit: shouldn't it be called `unresolved_paths` since it can accept a list of paths (as variable name suggests too)?

also: I've create a separate PR that persists this and filesystem: https://github.com/ray-project/ray/pull/50332